CMakeLists.txt

```diff
@@ -36,6 +37,23 @@ if (MSVC)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -wd4125")
 endif()
 
+# Enable ccache
+# LLVM_CCACHE_BUILD for consistency, but this currently has no effect
+set(LLVM_CCACHE_BUILD ${ONNX_MLIR_CCACHE_BUILD})
+if(ONNX_MLIR_CCACHE_BUILD)
+  find_program(CCACHE_PROGRAM ccache)
+  if(CCACHE_PROGRAM)
+    if(CMAKE_SYSTEM_NAME MATCHES "Windows")
+      set(CMAKE_C_COMPILER_LAUNCHER ${CCACHE_PROGRAM})
```

Comment:
Do the `_LAUCNHER` properties not work for Ubuntu? According to how they are defined, I would expect them to always work.

Reply:
They do, but RULE_LAUNCH_COMPILER is a bit more powerful, since ccache has some support for other compilers.

Reply:
Can you add a comment for this also?